infra/aws/terraform/prow-build-cluster/vpc.tf

```diff
@@ -21,20 +21,24 @@ limitations under the License.
 # VPC is IPv4/IPv6 Dual-Stack, but our cluster is IPv4 because EKS doesn't
 # support dual-stack yet.
 
+locals {
+  vpc_cidr = "10.0.0.0/8"
```

Comment:
We might want to leave _some_ room. How about `10.128.0.0/9`, or a /10 even?

Reply:
I think even `/12` should work. WDYT @ameukam @pkprzekwas?

Reply:
Let go with `10.128.0.0/10`.

Reply:
@ameukam Updated, PTAL.